package/scie-pants.lift.json

```diff
@@ -99,6 +99,18 @@
               "{scie.bindings.configure:PANTS_SHA_FIND_LINKS}"
             ]
           },
+          "pants-maybe-native-client": {
```

Comment:
I should have thought of this in the Pants-side PR, but if the native client binary could check `PANTS_NO_NATIVE_CLIENT` itself and immediately delegate then no new "pants-maybe-native-client" command would be needed. That would fit the spirit of the setting IIUC (almost no code executes to do that re-exec; so its not risky at all).

Reply:
The concern that I had (not sure whether it was founded) was that the native-client might be completely non-functional on someone's machine, and that they might need to completely bypass invoking it.

Reply:
Yeah, that seems pretty unlikely to me!

Reply:
The I think that I was primarily thinking about dynamic linking... we attempt to build wheels on fairly old machines to avoid those issues, but. And maybe it's unlikely to experience _new_ dynamic linking issues that we haven't already experienced via the native-engine module?

But yea, I could be convinced to go back and implement it on that side instead.

Reply:
Any, fwiw, I build the scie-pants binary, jump and ptex as true static binaries using libmusl to avoid this sort of thing completely. Pants might do the same if you're truly worried. But, again, probably fruitless because the handoff to the engine will hit a bunch of links that will then fail.